renku/service/controllers/templates_read_manifest.py

```diff
@@ -43,7 +48,27 @@ def context(self):
 
     def template_manifest(self):
         """Reads template manifest."""
-        template_manifest, _, _, _ = fetch_template(self.ctx["git_url"], self.ctx["ref"])
+        from PIL import Image
+
+        template_manifest, template_folder, _, _ = fetch_template(self.ctx["git_url"], self.ctx["ref"])
+
+        # NOTE: convert icons to base64
+        for template in template_manifest:
+            if "icon" not in template:
+                continue
+
+            # NOTE: prevent path traversal attack
+            icon_path = template_folder / (
+                (template_folder / template["icon"]).resolve().relative_to(template_folder.resolve())
```

Comment:
Shouldn't we resolve `template_folder` first?

Reply:
Actually it is already resolved in `fetch_template`, I've removed the `.resolve()` from template_folder